linkcheck/htmlutil/loginformsearch.py

```diff
@@ -54,5 +54,5 @@ def search_form(content, cgiuser, cgipassword):
             return form
 
     # not found
-    log.debug(LOG_CHECK, "Form with fields %s not found", ",".join(cginames))
+    log.warn(LOG_CHECK, "Form with fields %s not found", ",".join(cginames))
```

Comment:
isn't `warn` deprecated?

Reply:
It's our own:

https://github.com/linkchecker/linkchecker/blob/0f18c9b8f0784dbb0acf2d0867081df52d48bb36/linkcheck/log.py#L98-L105

logging.WARN doesn't generate deprecation warnings, which is just as well because we would have quite a bit to change, not just a simple search and replace